### PR TITLE
fix: give up on trying to check codepen.io links

### PR DESCRIPTION
**Pull request by wincent**
Since https://github.com/liferay/liferay-frontend-guidelines/pull/161 Cloudflare has been 503-ing these links.

They used to 403 them, and we got around that in https://github.com/liferay/liferay-frontend-guidelines/pull/134, but it looks like the party is over now, and I don't think it is worth trying to keep up with them on this one. [Another example of a PR](https://github.com/liferay/liferay-frontend-guidelines/pull/163) with a
spurious failure due to these links.

### CODE REVIEW

File: support/checkLinks.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ const statAsync = promisify(fs.stat);
 const FILTER_PATTERN = /\.md$/;
 
 /**
- * Need not-robot-like headers to prevent Cloudflare from rejecting our
- * external link checks with 403s.
+ * Use not-robot-like headers to attempt to prevent Cloudflare from
+ * rejecting our external link checks with 403 or 503 status.
```
Comment (wincent) [inline support/checkLinks.js:17]: Just to clarify why I'm not deleting this outright. There is still a chance that these headers *might* allow some Cloudflare-hosted links to be checked, because it seems the security levels are configurable on a per-site basis. For example, see the responses here:

- https://stackoverflow.com/a/18523671/2103996
- https://community.cloudflare.com/t/trying-to-curl-to-a-site-protected-by-cloudflare/32918

In short, depends on whether IPs are whitelisted, protections are turned on or off etc.

**Comment by wincent** [comment]
Too trivial to warrant review.
